rules/validator.go

```diff
@@ -88,6 +88,10 @@ func (v *validator) validateMappingRules(mrv map[string]*MappingRuleView) error
 
 		// Validate that the policies are valid.
 		for _, t := range types {
+			if len(view.Policies) == 0 {
```

Comment:
Probably should be done outside the for loop? Also might wanna move the policy validation logic into a `validatePolicies` function.